components/group/GroupRelatedInvitations.js

```diff
@@ -11,28 +11,30 @@ const RelatedInvitationRow = ({ item }) => (
 
 const GroupRelatedInvitations = ({ group, accessToken }) => {
   const groupId = group.id
+  const isV1Group = !group.domain
   const submissionName = group.details?.domain?.content?.submission_name?.value
 
   const [totalCount, setTotalCount] = useState(null)
 
   const loadRelatedInvitations = async (limit, offset) => {
-    const result = await api.getCombined(
+    const result = await api.get(
       '/invitations',
-      {
-        regex: `${groupId}/-/.*`,
-        expired: true,
-        type: 'all',
-        limit,
-        offset,
-      },
-      {
-        prefix: groupId.includes(submissionName) ? `${groupId}/.*` : `${groupId}/-/.*`,
-        expired: true,
-        type: 'all',
-        limit,
-        offset,
-      },
-      { accessToken }
+      isV1Group
```

Comment:
I think this would be easier to follow if this wasn't inside the argument list. I would declare a var 
```
const queryParam = isV1Group 
  ? { regex: `${groupId}/-/.*` }
  : { prefix: groupId.includes(submissionName) ? `${groupId}/.*` }
```
and then use that in the call.